src/search/index_manager.h

```diff
@@ -272,6 +273,62 @@ struct IndexManager {
 
     return Status::OK();
   }
-};
 
+  Status FieldValues(engine::Context &ctx, std::string_view index_name, std::string_view input_tag,
```

Comment:
We should iterate through the index data, instead of redis keys.

Reply:
@PragmaTwice Put in the fixes! thanks for the feedback :)